applications/IgaApplication/iga_application_variables.cpp

```diff
@@ -33,4 +33,7 @@ KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(POINT_LOAD)
 KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(LINE_LOAD)
 KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(SURFACE_LOAD)
 
+//Penalty Variables
+KRATOS_CREATE_VARIABLE(double, PENALTY_FACTOR)
```

Comment:
There is already a PENALTY parameters in the core, called INITIAL_PENALTY

Reply:
Thanks for the hint! INITIAL_PENALTY is wuite missleading don't you think so?